app/base.scss

```diff
@@ -0,0 +1,61 @@
+*{
+  padding: 0px;
+  width: 100%;
+  font-size: 1em;
+  font-weight: normal;
+  color: #ff4;
+
+  &:focus {
+    outline:0;
+  }
+}
+
+
+$brand-primary: #1087FF;
+$brand-secondary: #03E84B;
+
+$brand-active: #9B03E8;
+$brand-focus: #FFF71F
```

Comment:
**TODO FOR LOST POINTS**

You're missing a ``;`` after Line 18, and that's enough for your ``webpack-dev-server`` to break.